src/Appwrite/Platform/Tasks/ScheduleFunctions.php

```diff
@@ -84,8 +84,6 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
                     $queueForFunctions
                         ->setType('schedule')
                         ->setFunction($schedule['resource'])
-                        ->setMethod('POST')
-                        ->setPath('/')
```

Comment:
Scheduled functions should still have those. That is coming from CRON executions, when they can't set those. Those needs to remain.

Reply:
reverted